docs/business.yaml

```diff
@@ -2780,6 +2883,93 @@ paths:
             schema:
                 $ref: '#/components/schemas/Filing'
             examples:
+              incorporation-application-request:
+                summary: Incorporation Application Request
+                value:
+                  filing:
+                    header:
+                      certifiedBy: API Specs
+                      email: no_one@never.get
+                      name: incorporationApplication
+                      date: '{{today}}'
```

Comment:
replace this with mock value or any value as an example. Should not be `{{today}}`. Same for `nrNumber` on line 2899